server/aws-lsp-yaml-json/patchYamlJsonPackage.js

```diff
@@ -35,7 +35,7 @@ const filePathToPatchPathUnsafeEval = {
 function applyPatch(filePathToPatchPath) {
     for (var filePath in filePathToPatchPath) {
         const pathToPatch = `${__dirname}/${filePathToPatchPath[filePath]}`
-        const script = `cd ${rootPackage} && patch ${filePath} ${pathToPatch}`
+        const script = `cd "${rootPackage}" && patch "${filePath}" "${pathToPatch}"`
         console.log(script)
         const output = execSync(script, { encoding: 'utf-8', timeout: 2000 })
```

Comment:
It is almost always more robust to use the list form instead of passing a string to the exec() family of functions. That would avoid the quoting you have tried above, and avoids problems with wildcards and other shell-specific special chars that might appear in the `script` string.